be/src/column/column_pool.h

```diff
@@ -172,7 +172,11 @@ class CACHELINE_ALIGNED ColumnPool {
                 freed_bytes += release_column_if_large(_curr_free.ptrs[i], limit);
                 ASAN_POISON_MEMORY_REGION(_curr_free.ptrs[i], sizeof(T));
             }
-            _curr_free.bytes -= freed_bytes;
+            if (freed_bytes > 0) {
```

Comment:
why we need add this check? It seems freed_bytes was always greater or equal zero.

Reply:
```
size_t release_column_if_large(T* col, size_t limit) {
    auto old_usage = column_bytes(col);
    if (old_usage < limit) {
        return 0;
    }
    ...
```

Reply:
if freed_bytes = 0, means no large binary column, I will not exec  `tls_thread_status.mem_consume(freed_bytes);`